### PR TITLE
GH-2100, GH-2097: Onboarding test 2

### PR DESCRIPTION
**Pull request by leuryr**
Includes the addition of a new A/B test, which displays a different Hub layout on install for users.
- The A/B/C test for the Hub promo variant has been removed
- The ping param being used for the Hub promo variant has been repurposed with new values for Hub Layout test pings
- The A/B test timing bug fix from PR #601 has been implemented here as well

---
* [x] Have you followed the guidelines in [CONTRIBUTING.md](https://github.com/ghostery/ghostery-extension/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you added an explanation of what your changes do?
* [x] Does your submission pass tests?
* [x] Did you lint your code prior to submission?

### CODE REVIEW

**Comment by wlycdgr** [comment]
One bug I noticed: in the alternate Hub scenario, clicking on the Ghostery icon at the top of the sidebar makes the "this is the tab you're in" triangle indicator disappear

**Review by wlycdgr — COMMENTED** [review]
A few small things + we MAY need to keep the Metrics parts of the code for the old A/B test. Even if not, we should use a new Metrics query param name for this new test instead of `hp`, so we'll need a quick Metrics server ticket to add that param.

**Comment by wlycdgr** [comment]
NOTE: This PR should not be merged into develop until after 8.5.4 has been released.

File: src/background.js [hunk 1]
```diff
@@ -1759,10 +1758,10 @@ function initializeGhosteryModules() {
 			// We need to do this after running scheduledTasks for the first time
 			// because of an A/B test that determines which promo variant is shown in the Hub on install
 			if (globals.JUST_INSTALLED) {
-				const route = ((conf.hub_promo_variant === 'upgrade' || conf.hub_promo_variant === 'not_yet_set') && !IS_ANDROID) ? '' : '#home';
-				const showPremiumPromoModal = (conf.hub_promo_variant === 'midnight' && !IS_ANDROID);
+				const showAlternateHub = conf.hub_layout === 'alternate';
+				const route = conf.hub_layout === 'alternate' ? '#home' : '';
```
[the remark this answers is not in the document]
Comment (leuryr) [reply src/background.js]: Yep, definitely! Cleans the code up a bit.

File: app/hub/Views/SideNavigationView/SideNavigationView.jsx [hunk 2]
```diff
@@ -94,10 +98,12 @@ const SideNavigationView = (props) => {
 		disabled: disableNav,
 	});
 
+	const menuClassNames = ClassNames(`SideNavigation__menu ${!ah ? 'flex-child-grow' : ''} flex-container flex-dir-column`);
```
[the remark this answers is not in the document]
Comment (leuryr) [reply app/hub/Views/SideNavigationView/SideNavigationView.jsx]: Haha, for sure.

**Review by wlycdgr — APPROVED** [review]

**Review by christophertino — APPROVED** [review]

**Pull request merged**